experimental/single_unit/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
```

Comment:
Issue: Please either remove this from your PR or add it to the top-level Cargo.toml workspace

Reply:
I have added it to "components/icu/Cargo.toml"